.github/workflows/mingw.yml

```diff
@@ -0,0 +1,42 @@
+name: MinGW
```

Comment:
why do we need this? Is this not covered by our existing intel SDE tests?

Reply:
I'm not sure how this file got into this PR.  This appears to be a duplicate of CI tests we already have: https://github.com/aws/aws-lc/blob/main/.github/workflows/windows-alt.yml#L11

Reply:
It must have come in with an  intermediate merge somewhere along the way. I will remove it.